### ISSUE TITLE
Identity loss computed wrongly?

### ISSUE THREAD

**Issue by emilutz** [LynnHo/CycleGAN-Tensorflow-Simple#19]
`A2B_id_loss = identity_loss_fn(A, A2B)`

Here it seems the identity loss is computed between an instance of A and an instance of B. To my understanding the identity loss is supposed to check that an image from domain A is not altered by a generator towards domain A. In other words, it should be computed between G_B2A(A) and A

**Comment by LynnHo** [comment]
@emilutz Thank you！I will check that as soon as possible.

**Comment by LynnHo** [comment]
@emilutz You are right! I have corrected the mistake and updated the code from TensorFlow 2.0-alpha to TensorFlow 2.0. Thank you!
